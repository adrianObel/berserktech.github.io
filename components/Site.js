import React from 'react'
import MemberList from './MemberList'
import ContactUs from './ContactUs'
import Footer from './Footer'

const Site = ({ members }) => (
  <div className="pageContainer">
    <header>
      <div>
        <h1>BerserkTech</h1>
        <span>Software Developers Crew</span>
      </div>
    </header>
    <MemberList members={members} />
    <ContactUs />
    <Footer />
  </div>
)

export default Site
