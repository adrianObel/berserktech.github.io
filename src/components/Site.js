import React from 'react'
import MemberList from './MemberList'

const Site = ({ members }) => (
  <div className="pageContainer">
    <header>
      <div>
        <h1>BerserkTech</h1>
        <span>Software Developers Crew</span>
      </div>
    </header>
    <MemberList members={members} />
  </div>
)

export default Site
