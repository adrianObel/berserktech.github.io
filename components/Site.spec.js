import React from 'react'
import {mount} from 'enzyme'
import Site from './Site'

describe('<Site /> mount', () => {
  it('matches the snapshot', () => {
    const wrapper = mount(<Site members={[]} />)
    expect(wrapper).toMatchSnapshot();
  })
});