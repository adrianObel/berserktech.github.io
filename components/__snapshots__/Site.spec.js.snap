// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Site /> mount matches the snapshot 1`] = `
<div
  className="pageContainer"
>
  <header>
    <div>
      <h1>
        BerserkTech
      </h1>
      <span>
        Software Developers Crew
      </span>
    </div>
  </header>
  <section
    className="members"
  />
  <div
    className="contact-us"
  >
    <a
      href="mailto: ask@berserk.tech"
    >
      Contact Us Now
    </a>
  </div>
  <div
    className="footer"
  >
    <a
      href="https://github.com/berserktech"
      target="_blank"
    >
      github.com/berserktech
    </a>
    <a
      href="https://twitter.com/BerserkTech"
      target="_blank"
    >
      twitter.com/BerserkTech
    </a>
    <a
      href="https://medium.com/berserktech"
      target="_blank"
    >
      medium.com/BerserkTech
    </a>
    <a
      href="https://www.facebook.com/BerserkTech"
      target="_blank"
    >
      facebook.com/BerserkTech
    </a>
    <a
      href="https://medium.com/berserktech/manifesto-b9eda0fda1a7"
      style={
        Object {
          "color": "white",
          "flexBasis": "100%",
          "marginTop": "30px",
          "textAlign": "center",
        }
      }
      target="_blank"
    >
      <b>
        Read our Manifesto
      </b>
    </a>
  </div>
</div>
`;
